src/dodal/devices/aithre_lasershaping/goniometer.py

```diff
@@ -0,0 +1,15 @@
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.motor import Motor
+
+"""Goniometer and the stages it sits on"""
```

Comment:
Nit: This might be better as a docstring in the class as then sphinx can pick it up nicer